src/test/resources/serviceHearingValues.json

```diff
@@ -8,8 +8,8 @@
     "Good cause",
     "Rate of Assessment / Payability Issues - complex"
   ],
-  "hearingWindow": {
-    "hearingWindowDateRange": {
+  "shvHearingWindow": {
```

Comment:
Adding of 'svh' needs tor be reverted here.

Reply:
Resolved